src/ui/ui.css

```diff
@@ -20,8 +20,10 @@ body {
 }
 
 swg-loading-container {
-  position: absolute !important;
   width: 100% !important;
+  display: flex !important;
+  align-items: center !important;
+  justify-content: center !important;
   height: 148px !important;
```

Comment:
Check if this can be made `100%`, coupled with `min-height`.

Reply:
Done